README.md

```diff
@@ -17,104 +17,139 @@ This documentation outlines the necessary and optional [dependencies](#dependenc
+    conan install .. --build missing -pr:h=default -pr:b=default
+    conan build ..
+    ```
 
-```
-cd /path/to/code/build
-ctest .
-```
+    You can turn the various options on and off by adding flags to the `conan install` command, e.g.
+	The full list of build options can be found in the [conanfile](./conanfile.py).
 
-To install in directory `/X`, with libraries going to `X/' do:
+    ```bash
+    conan install .. --build missing -o cppflow=on -o openmp=on -o mpi=off -pr:h=default -pr:b=default
+    ```
 
-```
-cd /path/to/code/build
-cmake -DCMAKE_INSTALL_PREFIX=/X ..
-make install
-```
+Installing and building PURIFY with TensorFlow
+-------
+
+The `sopt` library includes an interface to TensorFlow for using trained models as priors in the Forward-Backward optimization algorithm. To build **PURIFY** with TensorFlow capability, some extra steps are currently required. We aim to simplify the build process in a future release.
+
+1. Install the [TensorFlow C API](https://www.tensorflow.org/install/lang_c)
+1. Clone the UCL fork of `cppflow` and create a `conan` package using
+
+    ``` bash
+    git clone git@github.com:UCL/cppflow.git
+    conan create /path/to/cppflow/ -pr:h=default -pr:b=default
+    ```
+1. Once the mandatory user-provided dependencies are present, `git clone` from the [GitHub repository](https://github.com/astro-informatics/purify):
+
+    ``` bash
+    git clone --recurse-submodules https://github.com/astro-informatics/purify.git
+    ```
+1. Create a `conan` package for `sopt` with the `cppflow` option set to "on"
+
+    ```bash
+    conan create /path/to/purify/sopt/ --build missing -s compiler.libcxx=libstdc++11 -o cppflow=on  -pr:h=default -pr:b=default
+    ```
+
+1. Then, the program can be built using `conan` with the `cppflow` option set to "on":
+
+    ``` bash
+    cd /path/to/purify
+    mkdir build
+    conan install .. --build missing -o cppflow=on -pr:h=default -pr:b=default
+    conan build ..
+    ```
 
 Testing
 -------
 
 To check everything went all right, run the test suite:
 
 ```
-cd /path/to/code/build
+cd /path/to/purify/build
 ctest .
 ```
 
 Usage
 ------
 
-The main purify executable lives either in the build directory or in the in the `bin` subdirectory
+The main `purify` executable lives either in the build directory or in the in the `bin` subdirectory
 of the installation directory. `purify` has one required argument, it a string for the file path of the config file containing the settings.
 
 `purify path/to/config.yaml`.
 
 A [template with a description of the settings](https://github.com/astro-informatics/purify/blob/development/data/config/config.yaml)
-is included in the `data/config` directory.`
+is included in the `data/config` directory.
 When `purify` runs a directory will be created, and the output images will be
 saved and time-stamped. Additionally, a config file with the settings used will
 be saved and time-stamped, helping for reproducibility and book-keeping.
 
 Docker
```

Comment:
Should we remove this section since we have not maintained the Docker image?

Reply:
My assumption was that if there is a self-contained dockerfile on dockerhub it might still work, hence the disclaimer I added below. I agree, create a separate issue and remove it from the documentation if the container is unusable.

Reply:
I guess this isn't relevant to this PR so I'll make an issue and we can leave it for now, then I think this one is ready to merge!